### ISSUE TITLE
Add more feedback for car chargers

### ISSUE THREAD

**Issue by frodeheg** [frodeheg/no.sparegris#20]
In the car charger section show:
* how much energy was Offered to/taken by the car 
* how much money you used on charging
* how much money you saved on delaying the charging to the cheapest hours ?

**Comment by dooniem** [comment]
Could the enhancement "new device, "Elbil Innsikt"" be added to this list?
With information about status, starting time, end time, remaining what-not. I understand a chart will be difficult in Homeys current "tile form". But this would be a quick window to just check from time to time, and after plugging in the car.

**Comment by frodeheg** [comment]
Yes I have been thinking about a device called "Charging controller"

**Comment by frodeheg** [comment]
Regarding
> how much money you saved on delaying the charging to the cheapest hours ?

This is a bit dubious... Money saved compared to what?
* Compared to charging at the most expensive hours?
* Compared to charging at the first hours in the plan?
* What about the power used? Compared to using all the power in the first hour or applying a max power per hour (the app don't know how much power the car would accept)?
* Then, how to spread out the power? If you had a budget of x kWh and the car didn't use it in an hour, then something else might have used it anyway, did you save anything then?

**Comment by dooniem** [comment]
Fine by me, it was your own suggestion it seems from reading the backlog 😅

**Comment by dooniem** [comment]
But if you were INSISTING on having some kind of calculator for savings. Here's some thoughts.
Monthly price pr. kWh is in the eye of the beholder.
It can either be the average of all the prices 1 month on Nord Pool, or it can be calculated through your Watt usage pr. hour and thus can vary from the average on Nord Pool.
Tibber advertise EV-charging through them save you at least 20% of YOUR average monthly price pr. kWh.
They handle this by putting the kWh used and the price paid for EV charging on a separate line on the bill.
So if the fluctuations of the price pr. kWh through one month is low, it's harder for Tibber to pick charge hours that will be 20% lower than your own average price. Hence they will have to pay you from their own pockets the difference.
But more often than not(?), the fluctuation is high, making a big difference charging the car on the cheapest hours and hence they rarely need to pay from their own pocket. 

So what some people are doing on their Homey's already, is looking at the total Watts charged by the EV-charger, and the actual mean price for that amount. Then comparing this to the "Household mean price" for the current/same period.
So their goal is to beat the 20% Tibber advertise. Because Tibber seems to "pocket" the difference if you actually save 30%. 

If your calculated percentage is below 20, you should regret not having Tibber as your supplier that month. Because they would've had to pay from their own pockets the difference between your actual % and 20 on your "EV bill".

Though it's a bit more complex if you add that Tibbers monthly fee is 39NOK to enable this "feature". I save that money each month by using Piggy instead.
